help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+---
+keywords: email;Email;e-mail;email destinations;salesforce;salesforce pardot;salesforce marketing cloud account engagement;salesforce pardot destination
+title: Salesforce Pardot
+description: The Salesforce Pardot (now renamed as Marketing Cloud Account Engagement) destination allows you to export your account data and activate it within Salesforce Pardot for your business needs.
+---
+# [!DNL Salesforce Pardot] connection
+
+[[!DNL Salesforce Pardot]](https://www.salesforce.com/products/marketing-cloud/overview/) *(now renamed as [!DNL Marketing Cloud Account Engagement])* allows you to capture and track leads by building templates, drag-and-drop builders; its powerful visual editor allows you to easily create custom landing pages, forms, and offers that attract potential leads for your sales teams. When someone engages with your forms, the lead is auto captured, and you get notified. Leads are also automatically scored and graded, so that those most likely to convert can be prioritized. Your forms can also trigger changes in lead scores upon conversion. [!DNL Salesforce Pardot] enables you to design lead tracks for all stages of the pipeline for targeted market segments and customer groups through email drip campaigns, lead management with nurturing and scoring and campaign segmentation.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Account Engagement]](https://developer.salesforce.com/docs/marketing/pardot/guide/overview.html) API, to work with leads for your sales needs after activating them within a new [!DNL Salesforce Pardot] segment.
+
+[!DNL Salesforce Pardot] uses OAuth 2 with Client Credentials as the authentication mechanism to communicate with the [!DNL Account Engagement] API. Instructions to authenticate to your [!DNL Salesforce Pardot] instance are further below, in the [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Salesforce Pardot] destination, here is a sample use case that Adobe Experience Platform customers can solve by using this destination.
+
+### Send emails to contacts for marketing campaigns {#use-case-send-emails}
+
+The marketing department of an online platform wants to broadcast an email based marketing campaign to a curated audience of B2B leads. The platform's marketing team can add new leads / update existing lead information through Adobe Experience Platform, build segments from their own offline data, and send these segments to [!DNL Salesforce Pardot], which can then be used to send the marketing campaign email.
+
+## Prerequisites {#prerequisites}
+
+Refer to the sections below for any prerequisites that have to be setup / information that is to be gathered before working with the [!DNL Salesforce Pardot] destination.
```

Comment:
Rephrasing suggestion to switch passive to active voice: 

Refer to the sections below for any prerequisites that you need to be set up in Experience Platform and Salesforce and for information that you need to gather before working with the [!DNL Salesforce Pardot] destination.